### DISCUSSION TITLE
Does Stripe use Markdoc for the API reference

### DISCUSSION THREAD

**Discussion by voxboy2** [markdoc/markdoc#50] [Q&A] [answered]
Hi guys, is MARKDOC used on this page https://stripe.com/docs/api/idempotent_requests and how can I handle it thanks?

**Comment by matv-stripe** [comment, marked as answer]
Right now we're only using it in our narrative docs (http://stripe.com/docs) but we have plans to use it in the reference docs as well! 

Currently, the reference docs are all custom; the frontend is React and the backend is Ruby. Our plan is to introduce Markdoc as an intermediary step to improve the authoring experience. The reference sites are interesting because a lot of the content is generated, so we are currently experimenting with ways to generate Markdoc content and make it editable.

**Reply by voxboy2** [reply]
Hi @matv-stripe  at what point does one need to customize Markdoc if he wants to use it . Currently this is our docs. docs.fincra.com

**Reply by matv-stripe** [reply]
@voxboy2 the interesting piece here is we currently use Markdoc for documentation that is all hand-written. A lot of API Reference documentation needs to be generated and match the API specification. We are currently exploring the best way to combine the great Authoring Experience of pure handwritten documentation with a good DX for adding new APIs
